crates/ruff_linter/src/rules/flake8_simplify/rules/yoda_conditions.rs

```diff
@@ -96,7 +95,7 @@ enum ConstantLikelihood {
 
 impl ConstantLikelihood {
     /// Determine the [`ConstantLikelihood`] of an expression.
-    fn from_expression(expr: &Expr, preview: PreviewMode) -> Self {
+    fn from_expression(expr: &Expr) -> Self {
```

Comment:
Should this now be `impl From<&Expr> for ConstantLikelihood {}`? (https://github.com/astral-sh/ruff/pull/9164#discussion_r1433002869)